### ISSUE TITLE
[QSelect][Safari] After select an option it scrolls to top of screen

### ISSUE THREAD

**Issue by robsontenorio** [quasarframework/quasar#13693]
### What happened?

When you select any option on QSelect it scrolls to top of screen.

### What did you expect to happen?

It should not scroll to top of screen

### Reproduction URL

https://jsfiddle.net/q8c1xvkb/

### How to reproduce?

REPRODUCTION 1

1. Scroll down and select an option 
2. After selecting an option it scrolls up to top of screen.

REPRODUCTION 2

1. Go to QSelect docs https://quasar.dev/vue-components/select#introduction
2. Then select any option on examples.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Safari

**Comment by bartduisters** [comment]
I tried the JSFiddle, the reference to the docs and in a freshly generated Vite project. I seem to have the expected outcome on Safari (Version 15.4 (17613.1.17.1.13)).

What version are you on?

![output](https://user-images.githubusercontent.com/43420049/174458602-8f9eefeb-f399-43cb-be81-6ea5f0943b65.gif)

**Comment by robsontenorio** [comment]
I am on macOS 13 beta developer.  

Maybe it is the issue. Thanks !

**Comment by bartduisters** [comment]
@robsontenorio Were you able to verify? If so, this issue can be closed.

**Comment by robsontenorio** [comment]
@bartduisters I can't verify because I can't go back to macOS 12 right now.

I am on macOS 13 beta2 and here it does not work.

![image](https://user-images.githubusercontent.com/118955/175604977-f71fa251-3c97-4918-a082-3e7bd3357e74.png)

**Comment by dehmlowm** [comment]
Sorry opened a dup(#14625 ) for this. I can confirm this happens on 16.0 on the docs page https://quasar.dev/vue-components/select

Also I noticed that it does not occur when using the  multiple tag.

**Comment by dehmlowm** [comment]
A few other Notes as I am trying to find a work around:
This also happens when tabbing to or from the qselect using _Tab_ or _Shift-Tab_. The q-select is properly focused but the page scrolls to the top. 
Also happens when clear an existing selection that is clearable. 
**If you use the Arrow keys to make a selection the press return or spacebar, it does _NOT_ scroll** Only occurs when using the mouse or tab key.

If you can look at the difference between making a selection using the arrow and enter keys and the click event that is likely the problem.

**Comment by dehmlowm** [comment]
Okay so a clunky work around is to use the multiple tag and limit the max values to one:
```
<q-select
    :options="['1', '2', '3']"
    multiple
    max-values="1"
    v-model="countArray"
    emit-value
  ></q-select
```

**Comment by pdanpdan** [comment]
It looks like ~IE11~ Safari has some problems with focusing 0 size elements :)

Add this to your css file until this fix is released

```css
.q-select__focus-target, .q-select__autocomplete-input {
  width: 1px;
  height: 1px;
}
```

Please tell me if it happens even with this fix (Safari on macos or ios)

**Comment by dehmlowm** [comment]
> It looks like ~IE11~ Safari has some problems with focusing 0 size elements :)
> 
> Add this to your css file until this fix is released
> 
> ```css
> .q-select__focus-target, .q-select__autocomplete-input {
>   width: 1px;
>   height: 1px;
> }
> ```
> 
> Please tell me if it happens even with this fix (Safari on macos or ios)

Added to app.scss and it behaves as expected thank you!!

**Comment by pdanpdan** [comment]
If someone has older versions of safari please test [this](https://github.com/quasarframework/quasar/issues/13693#issuecomment-1279477552) and let us know if there are problems.

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.10.1 & v1.21.1
